package.json

```diff
@@ -63,13 +63,13 @@
   },
   "dependencies": {
     "@kiali/k-charted-pf4": "0.3.0",
-    "@patternfly/patternfly": "2.4.1",
+    "@patternfly/patternfly": "2.36.0",
     "@patternfly/react-charts": "5.0.13",
-    "@patternfly/react-core": "3.89.2",
-    "@patternfly/react-styles": "3.5.16",
-    "@patternfly/react-table": "2.17.5",
-    "@patternfly/react-tokens": "2.6.21",
-    "@patternfly/react-virtualized-extension": "1.1.97",
+    "@patternfly/react-core": "3.104.0",
+    "@patternfly/react-styles": "3.5.22",
```

Comment:
I would update react-core to 3.112.1. It's the latest productivized release https://www.npmjs.com/package/@patternfly/react-core